__tests__/endpoints.test.js

```diff
@@ -91,3 +91,43 @@ describe("GET /api/articles/:article_id", () => {
     })
 })
 
+describe("/api/articles", () => {
+    test("200: returns object with all the articles", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+            const articles = body.body
+            
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                })
+            })
+        })
```

Comment:
This is a good test so far, but can give back a false pass when given an empty array, so to ensure this doesnt happen we need an assertion to ensure that the array is also not empty before entering the forEach.